system/iptables.py

```diff
@@ -335,6 +351,14 @@ def main():
     rule_is_present = check_present(iptables_path, module, module.params)
     should_be_present = (args['state'] == 'present')
 
+    # Check if we have both source and source_range
+    if module.params['source_range'] and module.params['source']:
```

Comment:
you don't need to do this check yourself, use the module function `mutually_exclusive`